Team.md

```diff
@@ -0,0 +1 @@
+Hier werden alle *Mitglieder* vorgestellt
```

Comment:
Sorry, ich habe vergessen aufzuschreiben, dass in der Kopfzeile immer folgendes stehen muss, damit es funktioniert:

```
---
layout: page
title: About
permalink: /about/
---
```

Füg das noch kurz hinzu (aber passe die Werte für `title` und `permalink` an) dann sollte es auch gehen.